prototype/src/main/webapp/upcoming-meetings.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>GoMeet Upcoming Meetings</title>
+  <script src="script.js"></script>
```

Comment:
Which script is this referring to? I remember telling Anna that script.js was a bad name, so this might have changed.

Reply:
I removed this line since the html doesn't actually refer to a javascript file.